portal-impl/src/com/liferay/portal/service.xml

```diff
@@ -289,21 +289,41 @@
 		<reference entity="User" package-path="com.liferay.portal" />
 		<reference entity="Website" package-path="com.liferay.portal" />
 	</entity>
-	<entity local-service="false" name="Country" remote-service="true">
+	<entity local-service="true" name="Country" remote-service="true" uuid="true">
 
 		<!-- PK fields -->
 
 		<column name="countryId" primary="true" type="long" />
 
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
 		<!-- Other fields -->
 
-		<column name="name" type="String" />
+		<column name="active" type="boolean" />
 		<column name="a2" type="String" />
 		<column name="a3" type="String" />
-		<column name="number" type="String" />
+		<column name="billingAllowed" type="boolean" />
```

Comment:
I think that `a2` and `a3` should be sorted before `active`. I'll double-check with Brian.